src/core/Akka.Streams.Tests/Akka.Streams.Tests.csproj

```diff
@@ -35,6 +34,12 @@
     <PackageReference Include="System.Runtime.Extensions" Version="4.3.1" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="xunit.runner.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Embedding `xunit.runner.json` directly inside the project folder to see if test raciness was caused by a missing setting file